### ISSUE TITLE
missing android platform

### ISSUE THREAD

**Issue by Delighting** [lukasoppermann/design-token-transformer#20]
design-token-transformer/config.json missing android support

**Comment by lukasoppermann** [comment]
Hey, we are working on adding more support, per chance android was actually the first one with examples added:
here are all transformers: https://github.com/lukasoppermann/design-token-transformer/tree/main/src

Here is the android example: https://github.com/lukasoppermann/design-token-transformer/tree/main/examples/android

**Comment by ninovanhooff** [comment]
Using this file as input, I expect to see an Android build output with colors for Android, like there is for ios and ios-swift.
Ideally I would like Android Jetpack Compose compatible files, but xml is fine, too.

But no android folder is created when following the readme.

- Export my colors from figma
- place the file in ./token/, overwriting the sample file
- npm run transform-tokens

When I then run the android task, it says it cannot find inputs for colors

```
 node ./examples/android/build.js

android
No properties for values/font_styles.xml. File not created.
No properties for values/dimens.xml. File not created.
No properties for values/colors.xml. File not created.
No properties for values-night/colors.xml. File not created.
Copying ./examples/android/filesToCopy/font_family.xml to ./examples/build/android/font/font_family.xml
```

Now the android folder is created, but as the cli output hints at, its only the template font file that gets created

**Comment by ninovanhooff** [comment]
Edit: had to create a new issue because this one is closed: #56
